SmartPlaces.Facilities/lib/OntologyMapper.Mapped/test/OntologyMapper.Mapped.Test.csproj

```diff
@@ -36,4 +36,10 @@
 	  <ProjectReference Include="..\src\OntologyMapper.Mapped.csproj" />
 	</ItemGroup>
 
+		<!-- Added Explicitly for GHSA-5crp-9r3c-p9vr - https://dev.azure.com/dynamicscrm/Solutions/_componentGovernance/ASPEN/alert/7074141?typeId=12185364 -->
+	<!-- Can be removed once Microsoft.Azure.Devices.Client fixes the vulnerability -->
```

Comment:
CopyPaste error. Vulerability is in `Microsoft.NET.Test.Sdk` & `Microsoft.AspNet.WebApi.Client`. This file has no references to `Microsoft.Azure.Devices.Client`.